sprouts.py

```diff
@@ -0,0 +1,246 @@
+import pygame
+from pygame import *
+import sys
+import time
+import random
+import math
+
+#Contains Game Settings: sproutset.py on Github
+from sproutset import Settings
+
+####OBJECTS - Will move later to .py or class
+#Game Window 
+pygame.init()
+ss = Settings()
+win = pygame.display.set_mode((ss.width, ss.height))
+pygame.display.set_caption('The Sprouts Game')
+
+running = True
+clock = pygame.time.Clock()
+
+#Player
+PlayerOne = 'Test1'
+PlayerOneScore = 0
+
+PlayerTwo = 'Test2'
+PlayerTwoScore = 0
+
+turn = 1 
+NumberStartingDots = 3
+
+#Dots
+dot_pos = []
+
+#Lines
+all_lines = []
+current_line = []
+
+####SPROUTS
+def sprouts():
+    global ss
+    #Start Screen
+    win.fill(ss.WHITE, None, 0)
+    text_writing('The Sprouts Game', ss.BLACK, 500, 25, 25) #Textwriting function
+    text_writing(f'{PlayerOne} score =' + str(PlayerOneScore), ss.BLACK, 125, 20, 12) #Updating Player Score text objects
```

Comment:
style guides I read suggest a maximum of 80 characters per line to increase readibility. however I don't know if comments/notes are included there but I would guess so. You did this in most parts of the code but here you could include a line break